Protos/api.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package api;
+
+service SchedulerApi {
+    rpc SubmitTask(TaskRequest) returns (TaskResponse) {}
+    rpc TaskCompleted(TaskCompletedRequest) returns (TaskCompletedResponse) {}
+    rpc RegisterWorker(RegisterWorkerRequest) returns (RegisterWorkerResponse) {}
+}
+
+service WorkerApi {
+    rpc GetResult(GetResultRequest) returns (GetResultResponse) {}
+    rpc ExecuteTask(TaskRequest) returns (TaskResponse) {}
```

Comment:
Include a SubmitTask API in worker stub as well?
This would be used by scheduler to send the request

Reply:
yes, this is what I meant here
https://github.com/krish8484/cs244b/pull/5#discussion_r1602189973
This SubmitTask API in worker would return future

Reply:
ExecuteTask and SubmitTask does the same action. I'll rename it for better clarity

Reply:
Modified in the latest commit